docs/platform/porting/kernel.md

```diff
@@ -102,50 +145,30 @@ If your platform is already loaded with the compatible boot loader software, you
 
 #### Boot loader setup and build
 
-To build the Tizen TM1 boot loader, follow these steps:
-
-1. Install and set up cross-compile tools on your system if the host has a different architecture than the target (such as x86).
-2. Start with cleaning up the `u-boot-tm1` source. Download the source from the [u-boot-tm1](https://review.tizen.org/git/?p=profile/mobile/platform/kernel/u-boot-tm1.git;a=summary) repository:
-   ```
-   $ make distclean
-   ```
-3. Set up the configuration for TM1.
-4. Build `u-boot`:
-   ```
-   $ make ARCH=arm
-   ```
-5. Once the build is successful, the `u-boot.bin` file is created. (This step is for preventing from flashing the other `u-boot.bin` file.):
-   ```
-   $ tools/mkimage_signed.sh u-boot.bin "tizen_tm1"
-   ```
+To build the Tizen Raspberry Pi 4 boot loader, refer to [U-Boot Build](#u-boot-build)
 
-   After the script is run, the `u-boot-mmc.bin` file is created.
-6. Create a boot loader tarball to download the `u-boot` binary onto the target:
-   ```
-   $ tar cvf bootloader.tar u-boot-mmc.bin
-   ```
-> [!NOTE]
-> Be careful when modifying the boot loader, incorrect configuration can damage the device permanently.
 
 #### Boot loader kernel parameters
 
-Command line parameters, such as the following example, can be passed from the boot loader to the [Linux](https://wiki.tizen.org/Linux) kernel:
+Command line parameters, such as the following example, can be passed from the boot loader to the Linux kernel:
 
 ```
-console=ttyS1,115200n8
-mem=1024M
-loglevel=1
+U-Boot> edit opts
+edit: loglevel=4
+U-Boot> saveenv
+Saving Environment to FAT... OK
+U-Boot>
 ```
 
 ### Kernel
 
-The kernel is the operating system that drives the platform. In this case, the kernel refers to the open-source [Linux](https://wiki.tizen.org/Linux) kernel that is customized for the Tizen platform. The following section gives a brief overview of the Tizen kernel setup, configuration, and the build procedure for building a Linux kernel for your Tizen platform. The output of the kernel binary is a uImage that is suitable only for a `u-boot` boot loader. If you have chosen a secure booting configuration in your boot loader, this uImage must be compatible with your boot loader.
+The kernel is the operating system that drives the platform. In this case, the kernel refers to the open-source Linux kernel that is customized for the Tizen platform. The following section gives a brief overview of the Tizen kernel setup, configuration, and the build procedure for building a Linux kernel for your Tizen platform. The output of the kernel binary is a uImage that is suitable only for a `u-boot` boot loader. If you have chosen a secure booting configuration in your boot loader, this uImage must be compatible with your boot loader.
```

Comment:
as far as I know, in Tizen RPI4, uImage is not used. so you can just skip to explain about "uImage" .